dotCMS/src/main/java/com/dotmarketing/common/reindex/BulkProcessorListener.java

```diff
@@ -95,6 +103,18 @@ public void afterBulk(final long executionId, final BulkRequest request, final B
         }
     }
 
+    static String getMatchingReservedIdIfAny(String id) {
+        String matchingReservedId = null;
+
+        for (String reservedId : RESERVED_IDS) {
+            if(id.contains(reservedId)) {
+                matchingReservedId = reservedId;
```

Comment:
Shouldn't we return/break the loop execution?

Reply:
Good catch! done